src/v2/Apps/Artwork/Components/ArtworkArtistSeries/ArtistSeriesArtworkRail.tsx

```diff
@@ -29,14 +37,48 @@ export const ArtistSeriesArtworkRail: React.FC<Props> = ({
 
   const isMobile = getENV("IS_MOBILE") === true
 
+  const trackArtworkClick = (artworkSlug, artworkID, index) => {
+    const properties: ClickedArtworkGroup = {
+      action: ActionType.clickedArtworkGroup,
+      context_module: ContextModule.moreFromThisSeries,
+      context_page_owner_type: OwnerType.artwork,
+      context_page_owner_slug: artwork.slug,
+      context_page_owner_id: artwork.internalID,
+      destination_page_owner_type: OwnerType.artwork,
+      destination_page_owner_id: artworkID,
+      destination_page_owner_slug: artworkSlug,
+      horizontal_slide_position: index,
+      type: "thumbnail",
+    }
+    trackEvent(properties)
+  }
+
+  const trackViewSeriesClick = () => {
+    const properties: ClickedArtworkGroup = {
+      action: ActionType.clickedArtworkGroup,
+      context_module: ContextModule.moreFromThisSeries,
+      context_page_owner_type: OwnerType.artwork,
+      context_page_owner_slug: artwork.slug,
+      context_page_owner_id: artwork.internalID,
+      destination_page_owner_type: OwnerType.artistSeries,
+      destination_page_owner_id: artistSeries.internalID,
+      destination_page_owner_slug: artistSeries.slug,
+      type: "viewAll",
```

Comment:
This link goes to an artist series, which makes me think it is not `clickedArtworkGroup` but `clickedArtistSeriesGroup`: https://cohesion.artsy.net/interfaces/_schema_events_click_.clickedartistseriesgroup.html

Reply:
This link is part of an element that is a group of artworks, so I believe this makes sense!

Reply:
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/2081340/91090950-e34dec00-e623-11ea-81ab-c8e0ddc01aed.png">

Reply:
Yea, from the [doc](https://docs.google.com/spreadsheets/d/1_VDAmRJz2K87FXDq7oh_tQSgwBv4syuZjaaR3sfetcI/edit#gid=644145883), it seems as though this would still be tracking the 'artwork shelf' clicks, even if the 'View' link goes to a non-artwork page.